Src/Sarif.PartnerMatcher.Function/HttpAnalyzeFn.cs

```diff
@@ -0,0 +1,48 @@
+using System;
```

Comment:
the folder name is called Partner instead of Pattern

Reply:
fixed